lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -165,12 +169,10 @@ internal class NextgenEngine(
         latencyTracker: NadelInternalLatencyTracker,
     ): ExecutionResult = supervisorScope {
         try {
-            val timer = NadelInstrumentationTimer(
-                ticker = latencyTracker::getInternalLatency,
-                instrumentation = instrumentation,
-                userContext = executionInput.context,
-                instrumentationState = instrumentationState,
-            )
+            val operationDefinition = queryDocument.getOperationDefinitionOrNull(executionInput.operationName)
+                ?: throw UnknownOperationException("Must provide operation name if query contains multiple operations")
```

Comment:
This is the exact same execution that `createExecutableNormalizedOperationWithRawVariables` throws

Refer to https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java#L417 and https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/language/NodeUtil.java#L74